site/content/en/docs/api/gwapi/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
to keep it consistent with `https://gateway.envoyproxy.io/docs/api/extension_types/`, should the dir name be `gateway_api` instead ?

Reply:
done